
Set total to not generate > 100% which overwrites random memory later on ...

  - Rene Rebe <rene@exactcode.de>

--- curl-7.13.0/src/main.c	2005-01-27 16:59:01.000000000 +0100
+++ curl-7.13.0-fixed/src/main.c	2005-03-03 19:53:02.000000000 +0100
@@ -2611,6 +2611,8 @@
   curl_off_t point = (curl_off_t)dlnow + (curl_off_t)ulnow +
     bar->initial_size; /* we've come this far */
 
+  if (total < point) total = point;
+
   bar->calls++; /* simply count invokes */
 
   if(total < 1) {
